requirements/README.md

```diff
@@ -0,0 +1,122 @@
+# Requirements
+DeepChem is a python deep-learning library for molecular machine learning. During its installation, it requires various prerequisite libraries to be installed. The various `.yml` files in the `requirements` folder contain the different libraries that are required based on the installer. The different folders are used to show the dependencies of the DeepChem library on the respective deep-learning backends.
```

Comment:
Can we remove this line because 1. we don't need what deepchem is about here (the first line) 2. it is kind of obvious that requirements folder contains requirements (third line).